index.html

```diff
@@ -22,9 +22,24 @@ <h2 class="info-note highlighted-text">You can check out all our events below!!<
     <h3 class="info-note"> To know how this page is getting developed, join our Discord Channel. Contact one of the existing members </h3>
   </main>
 
+  <div class="subscribe">
+      <div class="subscribe__heading">
+        <span>Want to stay updated?</span>
+        <span>Subscribe to  our Newsletter</span>
+      </div>
+      <div class="subscribe__inner">
+        <input class="subscribe__input" placeholder="Enter Email address" />
+        <button class="subscribe__button">Subscribe</button>
+      </div>
+      <div class="subscribe_status"></div>
+  </div>
   <footer>
     <p class="info-repo">The contents of this website are deployed from this <a href="https://github.com/Real-Dev-Squad/website" target="_blank" rel="noopener noreferrer">open sourced repo</a></p>
   </footer>
-
+  <script src="https://www.gstatic.com/firebasejs/7.17.0/firebase-app.js"></script>
+  <script src="https://www.gstatic.com/firebasejs/7.17.0/firebase-auth.js"></script>
+  <script src="https://www.gstatic.com/firebasejs/7.17.0/firebase-database.js"></script>
+  <script src="https://www.gstatic.com/firebasejs/7.17.1/firebase-analytics.js"></script>
```

Comment:
Let's move this code also to the drop-in file, since we are using firebase only for collecting emails right now